### ISSUE TITLE
Starting AppImage on Ubuntu 23.04 fails

### ISSUE THREAD

**Issue by IvoLimmen** [Daniele-rolli/Beaver-Notes#2]
Output when starting from command line:
```
Checking for update
[16413:0808/091103.187935:FATAL:gpu_data_manager_impl_private.cc(415)] GPU process isn't usable. Goodbye.
[1]    16413 trace trap (core dumped)  ./Beaver-notes-0.0.6.AppImage

```

OS:
Ubuntu 23.04
Uname -a: Linux QSD 6.2.0-26-generic #26-Ubuntu SMP PREEMPT_DYNAMIC Mon Jul 10 23:39:54 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux

Machine:
Dell Inc. XPS 15 9570
NVIDIA GeForce GTX 1050 Ti with Max-Q Design / Intel® UHD Graphics 630 (CFL GT2)

Windowing system: Wayland
GNOME version: 44.3

**Comment by Daniele-rolli** [comment]
Thanks for your feedback. I've had a similar problem with a Radeon GPU. Considering your XPS has an Nvidia GPU, you might want to check your GPU driver or try starting the appimage using the integrated graphics. Otherwise, if you feel comfortable using snap packages, give it a try.

**Comment by Daniele-rolli** [comment]
The "GPU process isn't usable" issue in linux based system has been resolved in version 0.0.6-a.appimage
